tests/apiv2/observables.py

```diff
@@ -44,6 +76,23 @@ def test_observable_search(self):
         data = response.json()
         self.assertEqual(len(data), 2)
 
+    def test_observable_search_extended_response(self):
```

Comment:
This tests the same sort of thing than `test_observable_search`. Maybe you'd want to search on one of the extended attributes instead of value?

Reply:
I added a dedicated test because the already existing one checks the query matches two observables. I could replace hostname with two files named empty and empty2 if you think that makes more sense.